core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListManager.java

```diff
@@ -255,6 +256,17 @@ public void processMasternodeListDiff(SimplifiedMasternodeListDiff mnlistdiff, b
             unCache();
             failedAttempts = 0;
 
+            if (!isQuorumRotationEnabled) {
+                quorumList.forEachQuorum(true, new SimplifiedQuorumList.ForeachQuorumCallback() {
+                    @Override
+                    public void processQuorum(FinalCommitment finalCommitment) {
+                        if (finalCommitment.getLlmqType() == params.getLlmqTypeInstantSendDIP24().getValue()) {
+                            isQuorumRotationEnabled = true;
+                        }
+                    }
+                });
+            }
```

Comment:
This code determines if DIP24 is active.  We look for quorums with DIP24 quorum types.